assets/src/scss/blocks.scss

```diff
@@ -4,3 +4,4 @@
 @import "blocks/Accordion/AccordionStyle";
 @import "blocks/Cookies/CookiesStyle";
 @import "blocks/Counter/CounterStyle";
+@import "blocks/Submenu/Submenu.scss";
```

Comment:
We should rename this file `SubmenuStyle.scss` to be consistent with other blocks